<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="techtrail-HP-ProBook-640-G2" failures="0" tests="4" name="Auth_Services.TC_6_Invalid_Scenario_SendOTP" time="0.931" errors="0" timestamp="2023-05-23T10:22:49 IST" skipped="0">
  <testcase classname="Auth_Services.TC_6_Invalid_Scenario_SendOTP" name="Invalid_otp" time="0.660"/>
  <system-out/>
  <testcase classname="Auth_Services.TC_6_Invalid_Scenario_SendOTP" name="Invalid_otp" time="0.093"/>
  <system-out/>
  <testcase classname="Auth_Services.TC_6_Invalid_Scenario_SendOTP" name="Invalid_otp" time="0.105"/>
  <system-out/>
  <testcase classname="Auth_Services.TC_6_Invalid_Scenario_SendOTP" name="Invalid_otp" time="0.073"/>
  <system-out/>
</testsuite> <!-- Auth_Services.TC_6_Invalid_Scenario_SendOTP -->
